exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillDatabaseMetaDataImpl.java

```diff
@@ -219,11 +226,26 @@ public boolean storesMixedCaseQuotedIdentifiers() throws SQLException {
     return super.storesMixedCaseQuotedIdentifiers();
   }
 
-  // TODO(DRILL-3510):  Update when Drill accepts standard SQL's double quote.
   @Override
   public String getIdentifierQuoteString() throws SQLException {
     throwIfClosed();
-    return "`";
+    boolean systemOption = false;
+    boolean sessionOption = false;
+    String sql = "select type, bool_val from sys.options where name = 'parser.ansi_quotes'";
+    ResultSet rs = executeSql(sql);
+    while (rs.next()) {
+      if (rs.getString(1).equals("SYSTEM")) {
+        systemOption = rs.getBoolean(2);
+      }
+      if (rs.getString(1).equals("SESSION")) {
+        sessionOption = rs.getBoolean(2);
+      }
+    }
+    if (systemOption || sessionOption) {
```

Comment:
If session option is turned off, but system options is turned on for ansi_quote, do we still return DOULBE_QUOTE?